src/app/core/parsers/zod-entity-parser.service.ts

```diff
@@ -155,4 +163,36 @@ export class ZodEntityParser implements EntityParser {
   public parseShowStyleVariant(showStyleVariant: unknown): ShowStyleVariant {
     return this.showStyleVariantParser.parse(showStyleVariant)
   }
+
+  private readonly studioConfigurationParser = zod.object({
+    settings: zod.object({
+      mediaPreviewUrl: zod.string().startsWith('http://', 'Media preview url must start with http://').and(zod.string().min(9, 'Media preview url must more than 9 characters long')),
+    }),
+    blueprintConfiguration: zod.object({
+      ServerPostrollDuration: zod.number().min(0, 'Server postroll duration must be 0 or more'),
```

Comment:
It would be nice if the sentences in messages we provide are ended with a period.